### PR TITLE
FB and Ln Social Auth 

### PR DESCRIPTION
**Pull request by bygirish**
This PR includes -
- Created UI for FB and Ln Social Buttons
- API callings done

Bug Fixes
- Fixed Button click event

### CODE REVIEW

File: app/presentation/auth/SocialAuth/index.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import React from "react";
+
+const SocialAuthFB = () => {
+
+    return (
+        <div>
+            {'FB'}
```
Comment (Akshaykumar9902) [inline app/presentation/auth/SocialAuth/index.tsx]: Please complete FB component UI

File: app/presentation/auth/SocialAuth/index.tsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        </div>
+    )
+}
+
+const SocialAuthLn = () => {
+
+    return (
+        <div>
+            {'Ln'}
```
Comment (Akshaykumar9902) [inline app/presentation/auth/SocialAuth/index.tsx]: Please Complete Ln Component UI.

**Review by Akshaykumar9902 — CHANGES_REQUESTED** [review]
Please connect me if you facing any issues.

File: app/presentation/auth/SocialAuth/index.tsx (hunk 1 above)
Comment (bygirish) [reply app/presentation/auth/SocialAuth/index.tsx]: Done

File: app/presentation/auth/SocialAuth/index.tsx (hunk 2 above)
Comment (bygirish) [reply app/presentation/auth/SocialAuth/index.tsx]: Done

File: app/presentation/auth/SocialAuth/index.tsx [hunk 3]
```diff
@@ -0,0 +1,24 @@
+import React from "react";
+
+const SocialAuthFB = () => {
+
+    return (
+        <div>
+            {'Complete FB UI'}
```
Comment (Akshaykumar9902) [inline app/presentation/auth/SocialAuth/index.tsx]: This is not the right way.
Please complete the task as the discussion we had.

**Review by Akshaykumar9902 — CHANGES_REQUESTED** [review]

**Review by Akshaykumar9902 — APPROVED** [review]

**Pull request merged**